sense_api.py

```diff
@@ -67,67 +64,92 @@ def get_usage(s, sense_monitor_id, start ,granularity): # NOT WORKING
+        response = self.s.get(API_URL+'monitors/%s/devices' % self.sense_monitor_id, headers=self.headers)
         return response.json()
 
 
     def always_on_info(self):
         # Always on info - pretty generic similar to the web page
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/app/monitors/%s/devices/always_on' % self.sense_monitor_id, headers=self.headers)
+        response = self.s.get(API_URL+'app/monitors/%s/devices/always_on' % self.sense_monitor_id, headers=self.headers)
         return response.json()
 
 
     def get_monitor_info(self):
         # View info on your monitor & device detection status
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/app/monitors/%s/status' % self.sense_monitor_id, headers=self.headers)
+        response = self.s.get(API_URL+'app/monitors/%s/status' % self.sense_monitor_id, headers=self.headers)
         return response.json()
 
 
     def get_device_info(self, device_id):
         # Get specific informaton about a device
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/app/monitors/%s/devices/%s' % (self.sense_monitor_id, self.device_id), headers=self.headers)
+        response = self.s.get(API_URL+'app/monitors/%s/devices/%s' % (self.sense_monitor_id, self.device_id), headers=self.headers)
         return response.json()
 
 
     def get_notification_preferences(self):
         # Get notification preferences
         payload = {'monitor_id': '%s' % self.sense_monitor_id}
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/users/%s/notifications' % self.sense_user_id, headers=self.headers, data=payload)
+        response = self.s.get(API_URL+'users/%s/notifications' % self.sense_user_id, headers=self.headers, data=payload)
         return response.json()
 
 
     def get_daily_kWh(self):
         payload = {'n_items': 30}
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/users/%s/timeline' % self.sense_user_id, headers=self.headers, data=payload)
+        response = self.s.get(API_URL+'users/%s/timeline' % self.sense_user_id, headers=self.headers, data=payload)
         return response.json()['items'][1]['body']
 
 
     def get_all_usage_data(self):
         payload = {'n_items': 30}
         # lots of info in here to be parsed out
-        response = self.s.get('https://api.sense.com/apiservice/api/v1/users/%s/timeline' % self.sense_user_id, headers=self.headers, data=payload)
+        response = self.s.get(API_URL+'users/%s/timeline' % self.sense_user_id, headers=self.headers, data=payload)
         return response.json()
 
 
-
-def main():
-    #foo = sensenable()
+if __name__ == "__main__":
+    import pprint
+    import getpass
+    # collect authn data
+    username = raw_input("Please enter you Sense username (email address): ")
+    password = getpass.getpass("Please enter your Sense password: ")
+    sense = Sensenable(username, password)
```

Comment:
Hah, I actually meant to name it Senseable. Still using old name here `sense = Senseable(username, password)`